sphinx/writers/latex.py

```diff
@@ -275,6 +275,22 @@ def escape_abbr(text):
     return re.sub('\.(?=\s|$)', '.\\@', text)
 
 
+def rstdim_to_latexdim(width_str):
+    """Convert `width_str` with rst length to LaTeX length."""
+    match = re.match('^([0-9.]+)\s*(\S*)$', width_str)
+    if not match:
+        raise ValueError
+    res = width_str
+    amount, unit = match.groups()[:2]
+    if unit in ('', "px"):
+        res = "%s\\sphinxpxdimen" % amount
```

Comment:
memo: the reason for using ``px`` as default unit is that this is what has been so far documented by Sphinx for all targets, when no unit is specified.
http://www.sphinx-doc.org/en/1.4.4/rest.html#images

So far LaTeX was considered as not supporting ``px`` but PDFLaTeX for at least ten years natively accepts it. To maintain multi-target coherence, using default ``96px=1in`` (or ``4px=3bp``) seems the thing to do (thus Sphinx changes pdflatex default setting of ``72px=1in``, or ``1px=1bp`` where ``1bp=1/72in``)